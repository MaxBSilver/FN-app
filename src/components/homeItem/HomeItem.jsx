import React from 'react'
import moment from 'moment';
export default function HomeItem(props) {
    console.log(props)
    console.log(props.time)
    return (
        <div>
            <h1>{props.title}</h1>
            <img alt={props.title} src={props.image}></img> 
            <p>{props.body}</p>
            <span>{moment(props.time).format('MM/DD/YYYY')}</span>
        </div>
    )
}
