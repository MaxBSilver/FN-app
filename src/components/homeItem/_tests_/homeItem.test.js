import React from "react";
import { shallow } from "enzyme";
import HomeItem from "../HomeItem";

describe("HomeItem", () => {
  let wrapper;

  beforeEach(() => {
    const mockProps = {
      meta: { mainColor: "#cd8823", adSpace: null },
      time: 1559578127,
      body: "The Eternal Struggle Set is available now!",
      image:
        "https://fortnite-public-files.theapinetwork.com/91bdf9d0a4caa1fe8",
      title: "Darkness or Light"
    };
    wrapper = shallow(<HomeItem {...mockProps} />);
  });

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
