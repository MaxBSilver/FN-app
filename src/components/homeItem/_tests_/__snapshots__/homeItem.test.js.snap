// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeItem should match snapshot 1`] = `ShallowWrapper {}`;
